directory/kv.go

```diff
@@ -0,0 +1,43 @@
+package directory
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/wakeful-deployment/operator/service"
+	"strings"
+)
+
+type KV struct {
+	Key         string
+	Value       string
+	ModifyIndex int
+	Service     service.Service
```

Comment:
Let's remove this property and just have `DecodeService` return it. I don't think we have a need to cache this anymore.